astro/src/content/docs/get-started/core-concepts/tenants.mdx

```diff
@@ -173,8 +184,34 @@ Control Email and Phone Number identity verification settings from one place.
   <APIField name="Verification complete template" optional since="1.99.9">
     The email or message template to use when notifying a user that their email address or phone number has been verified.
   </APIField>
+  <APIField name="Verification strategy" optional since="1.99.9">
```

Comment:
Also versioning on these fields. I think some of these were updated when we had a separate tab for phone configuration. Should they go back now that it also refers to the email fields?

Reply:
Missed this, yeah they probably should go back.